libursa/src/errors/mod.rs

```diff
@@ -67,8 +72,12 @@ impl Fail for UrsaCryptoError {
 
 impl UrsaCryptoError {
     pub fn from_msg<D>(kind: UrsaCryptoErrorKind, msg: D) -> UrsaCryptoError
-        where D: fmt::Display + fmt::Debug + Send + Sync + 'static {
-        UrsaCryptoError { inner: Context::new(msg).context(kind) }
+    where
+        D: fmt::Display + fmt::Debug + Send + Sync + 'static,
+    {
+        UrsaCryptoError {
+            inner: Context::new(msg).context(kind),
+        }
```

Comment:
I just discovered the ['error-chain' crate](https://crates.io/crates/error-chain) the other day when I was hacking on BetterSign. It might also help with the custom error types. Not criticism, just a suggestion.

Reply:
I'll look into it